libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -341,6 +345,7 @@ void GCS_MAVLINK::send_proximity() const
 
     const uint16_t dist_min = (uint16_t)(proximity->distance_min() * 100.0f); // minimum distance the sensor can measure in centimeters
     const uint16_t dist_max = (uint16_t)(proximity->distance_max() * 100.0f); // maximum distance the sensor can measure in centimeters
+    const float sensor_orientation[4] = { 0.0f }; // TODO: populate the Quaternion
```

Comment:
AFAIK you don't need the bit in the braces here; older compilers needed it but the standard doesn't.